nixos/modules/services/display-managers/greetd.nix

```diff
@@ -0,0 +1,104 @@
+{ config, lib, pkgs, ... }:
+with lib;
+
+let
+  cfg = config.services.greetd;
+  tty = "tty${toString cfg.vt}";
+  settingsFormat = pkgs.formats.toml {};
+in
+{
+  options.services.greetd = {
+    enable = mkEnableOption "greetd";
+
+    package = mkOption {
+      type = types.package;
+      default = pkgs.greetd;
```

Comment:
```suggestion
      default = pkgs.greetd.greetd;
```

Reply:
Thanks for catching that! My own greetd-package was still active while testing the module.